src/styles/blocks/header.scss

```
@@ -0,0 +1,20 @@
+.header{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__img{
+    max-width: 100%;
+  }
+
+}
```

Write code in 1 style
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
  text-transform: uppercase;
  font-weight: 500;
  font-size: 12px;
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);

  &__logo {
    width: 40px;
    height: 40px;
  }

  &__img {
    max-width: 100%;
  }
}
```